src/combinatorics/simple_graph/basic.lean

```diff
@@ -118,6 +119,11 @@ lemma adj_comm (u v : V) : G.adj u v ↔ G.adj v u := ⟨λ x, G.symm x, λ x, G
 
 @[symm] lemma adj_symm {u v : V} (h : G.adj u v) : G.adj v u := G.symm h
 
+def support : set V := rel.image G.adj set.univ
+
+lemma mem_support {v : V} : v ∈ G.support ↔ ∃ w, G.adj v w :=
+  by simp [support, rel.mem_image, adj_comm]
+
```

Comment:
This might be better-placed right after `end order`

Make sure to add a docstring for `def support`